pkg/k8shandler/deployment.go

```diff
@@ -32,7 +32,7 @@ func (node *deploymentNode) isDifferent(cfg *desiredNodeState) (bool, error) {
 	for _, container := range node.resource.Spec.Template.Spec.Containers {
 		if container.Name == "elasticsearch" {
 			if container.Image != cfg.ESNodeSpec.Spec.Image {
-				logrus.Debugf("Container image '%v' is different that desired, updating..", container.Image)
+				logrus.Debugf("Container image '%v' is different that desired", container.Image)
```

Comment:
To me, it would be more useful to know what node/deployment needs to be updated rather than which image